### ISSUE TITLE
make the game more friendly with shop egg

### ISSUE THREAD

**Issue by Popoteguitou** [pokeclicker-dev/pokeclicker#170]
those who did not know pokemons may have some troubles finding how to fill their dex... nothing say how to catch bulbasaur! there may be some player who didn't even know what type are those missing pkmn in their dex! 
I thinked of two way to make it easier:
1) show uncaugh pkmn type in pokedex, so player know which egg try
2) show which pkmn can be in which egg...
for 2), it may be mom return ! when you beat the elite 4 of a region, you may unlock a "mom" option in pallet town whose show you which pkmn are obtainable in which egg.

**Comment by elitepokefans** [comment]
It may be good but I doubt anyone has no idea of pokemon type.  The franchiase of Pokemon did exist since maybe from 1998?  Everyone probably grew up with Pokemon.  And how do they even come across this game?  They may have search through the internet for it or hear from friends and family?

**Comment by Popoteguitou** [comment]
yeah, but who remember Houndor dual type? not those who only play the last gens...

**Comment by RedSparr0w** [comment]
I don't feel like every aspect of the game needs to be right there available, sometimes it is better to seek the info out from other people, or a wiki or similar.
